src/components/App/App.tsx

```diff
@@ -38,6 +39,19 @@ const App = () => {
     password, setPassword, bio, setBio, isLoggedIn, setIsLoggedIn
   };
 
+  useEffect(() => {
+    const script = document.createElement('script');
+
+    script.src = "https://maps.googleapis.com/maps/api/js?key=AIzaSyBS_2C_7FuKpDZ6Hv4uo2Shqn_fctik4Ik&libraries=places";
+    script.async = true;
+
+    document.body.appendChild(script);
+
+    return () => {
+      document.body.removeChild(script);
+    }
+  }, []);
+
```

Comment:
I'd love to talk through this part of the code

Reply:
Yes me too!